### ISSUE TITLE
Consider getting rid of forks in config files

### ISSUE THREAD

**Issue by gumb0** [ethereum/retesteth#67]
I don't really get what's the purpose of `forks` (and `additonalForks`) in config files. 

They look all identical to me in all of the currently existing client configs (in tests repo). You could just hard-code them in retesteth (or have a single config file for retesteth), that would simplify everything for a user and reduce maintenance pain 6x.

Is it to adapt to the possibility of forks having different names in different clients? I think in reality all the clients have some adaptors in their code to integrate retesteth support, so I don't think it's really an issue (currently aleth, geth and besu all work with the same genesis config jsons, don't they?)

~~Is it to be able to run the subset of forks for each test? This should be better tuned with command line arguments to retesteth.~~ No, it fails in case not all of the forks requited for tests are listed.

To summarize, it seems that having *.json config files should be enough, you don't really need to repeat their names in `config` file.

**Comment by winsvega** [comment]
`forks` define the progression order in which a test case would be generated on.
in the test you set  `>=Frontier`   and it will be decoded using `Forks` section into `Frontier, Homestead, EIP150... ConstantinopleFix`
`additional forks` section defines all other allowed to run on the client forks.
